ai_diffusion/workflow.py

```diff
@@ -423,7 +473,9 @@ def generate(
     batch = 1 if live.is_active else batch
 
     w = ComfyWorkflow(comfy.nodes_inputs)
-    model, clip, vae = load_model_with_lora(w, comfy, style, is_live=live.is_active)
+    model, clip, vae = load_model_with_lora(
+        w, comfy, style, is_live=live.is_active, additional_loras=_parse_loras(comfy, cond.prompt)
```

Comment:
Since this is repeated a lot I would prefer if the prompt was passed to `load_model_with_lora` directly, non-optional (I don't think we have a case where we don't want to support in-prompt loras?)

Then parse loras inside the function.